docs/zh_cn/understand_mmcv/data_transform.md

```diff
@@ -192,15 +192,15 @@ pipeline = [
                 dict(type='Rotate', angle=30)
             ],  # 第二种随机变换组合
         ],
-        pipeline_probs=[0.4, 0.6]  # 两种随机变换组合各自的选用概率
+        prob=[0.4, 0.6]  # 两种随机变换组合各自的选用概率
         )
     ...
 ]
 ```
 
 ### 多目标扩展（ApplyToMultiple）
 
-通常，一个数据变换类只会从一个固定的字段读取操作目标。虽然我们也可以使用 `Remap` 来改变读取的字段，但无法将变换一次性应用于多个字段的数据。为了实现这一功能，我们需要借助多目标扩展包装（`ApplyToMultiple`）。
+通常，一个数据变换类只会从一个固定的字段读取操作目标。虽然我们也可以使用 `ApplyToMapped` 来改变读取的字段，但无法将变换一次性应用于多个字段的数据。为了实现这一功能，我们需要借助多目标扩展包装（`ApplyToMultiple`）。
 
 多目标扩展包装（`ApplyToMultiple`）有两个用法，一是将数据变换作用于指定的多个字段，二是将数据变换作用于某个字段下的一组目标中。
```

Comment:
Proposing `TransformBrocaster` here. The role of this transform also seems clearer though the name is a bit hard to pronounce. (I find that ROS has approved this name in their library http://docs.ros.org/en/hydro/api/tf/html/c++/classtf_1_1TransformBroadcaster.html)

Reply:
@gaotongxiao @ZwwWayne 
Which one would be better, `KeyMapper` and `TransformBroadcaster` or `KeyMap` and `TransformBroadcast`？ Do we have a convention about transform names being verbal or nominal?

Reply:
Adapting a different name convention for transform wrappers may help users tell them apart from other transforms, so I'd vote for the names ending with "-er". Also, `RandomChoice` -> `RandomChooser`

Reply:
> Using a different name convention for transform wrappers may help users tell them apart from other transforms, so I'd vote for the names ending with "-er". Also, `RandomChoice` -> `RandomChooser`

But `RandomChoice` is align with [`RandomChoice` in torchvision](https://pytorch.org/vision/stable/generated/torchvision.transforms.RandomChoice.html#torchvision.transforms.RandomChoice)

Reply:
I see, we can stick to `RandomChoice`